pyre.py

```diff
@@ -47,9 +51,46 @@ def color(r, g, b):
     assert(len(self.particles) == self.NUM_PARTICLES)
 
     self.resize()
+    self.volume = 1.0
+    if pygame_available:
```

Comment:
not necessary for this commit, but do you think we should move the audio checking logic into `play_fire`? might be cleaner. thoughts?